src/app/globals.css

```diff
@@ -25,3 +25,42 @@ body {
     )
     rgb(var(--background-start-rgb));
 }
+
+.h1-landing-text {
+  /* TEMPORARY color till global theme is placed*/
+  color: #035496;
+  font-size: 2em;
+  margin-bottom: 1rem;
+  margin-left: 1em;
+  margin-right: 1em;
+}
+
+.h2-landing-text {
+  /* TEMPORARY color till global theme is placed*/
+  color: #057cde;
+  font-size: 1.5em;
+  margin-bottom: 0.5rem;
+  margin-left: 1em;
+  margin-right: 1em;
+}
+
+.p-landing-text {
```

Comment:
[Comment]: This conflates two ideas, the HTML tag (`p`) and semantic meaning (`landing-text`) using a CSS class - you would generally either do one or the other. I would lean towards styling by tag (e.g. `p`) and only use a class for exceptions to default rules.

No need to change anything here; I'll handle this as part of global styling.